src/lib.rs

```diff
@@ -805,6 +819,29 @@ impl<K: Hash + Eq, V, S: BuildHasher, A: Clone + Allocator> LruCache<K, V, S, A>
         self.cap = cap;
     }
 
+    pub fn evict_by_epoch(&mut self, epoch: Epoch) {
```

Comment:
some RW's streaming executors assume that the current epoch's element can not be evicted. so we might need some check when the `evict_epoch` is higher than the `current_epoch`. An elegant method is to store the evicted epoch in Self and lazily delete it when `update_epoch`. but we can decide it later. because the behavior can maintained by the called of `evict_by_epoch`

Reply:
In RW, `evict_by_epoch` will only be called at the end of an epoch.